nbconvert/exporters/templateexporter.py

```diff
@@ -163,7 +163,39 @@ def _template_file_default(self):
     
     #Extension that the template files use.
     template_extension = Unicode(".tpl").tag(config=True, affects_environment=True)
+    
+    exclude_input = Bool(False,
+        help = "This allows you to exclude code cell inputs from all templates if set to True."
+        ).tag(config=True)
+
+    exclude_input_prompt = Bool(False,
+        help = "This allows you to exclude input prompts from all templates if set to True."
+        ).tag(config=True)
+
+    exclude_output = Bool(False,
+        help = "This allows you to exclude code cell outputs from all templates if set to True."
+        ).tag(config=True)
+
+    exclude_output_prompt = Bool(False,
+        help = "This allows you to exclude output prompts from all templates if set to True."
+        ).tag(config=True)
+
+    exclude_code = Bool(False,
```

Comment:
I only changed code rather than anything else as there weren't other ambiguities and this way the options are shorter to invoke.